Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/BocListValidationResultDispatchingValidatorTest.cs

```diff
@@ -170,5 +170,68 @@ public void RefreshErrorMessage_WithInternalAndRowFailures_ReturnsAppendedMessag
 
       Assert.That(validator.ErrorMessage, Is.EqualTo("Errors on BocList\r\nErrors on rows in BocList"));
     }
+
+    [Test]
+    public void EvaluateIsValid_WithoutFailures_EmptyErrorMessageAndValid ()
+    {
+      var validationFailureRepoStub = new Mock<IBocListValidationFailureRepository>();
+      validationFailureRepoStub.Setup(_ => _.GetUnhandledValidationFailuresForBocListAndContainingDataRowsAndDataCells(false))
+          .Returns(Array.Empty<BocListValidationFailureWithLocationInformation>());
+
+      var error = "BocList has validation errors.";
+      var resourceManagerStub = new Mock<IResourceManager>();
+      resourceManagerStub.Setup(_ => _.TryGetString("Remotion.ObjectBinding.Web.UI.Controls.BocList.RemainingValidationFailureText", out error)).Returns(true);
+
+      var bocListMock = new Mock<Control> { CallBase = true };
+      bocListMock.As<IBocList>().Setup(_ => _.ValidationFailureRepository).Returns(validationFailureRepoStub.Object);
+      bocListMock.As<IBocList>().Setup(_ => _.GetResourceManager()).Returns(resourceManagerStub.Object);
+      bocListMock.Object.ID = "someBocList";
```

Comment:
We don't need the resource manage setups for this test if I am not mistaken?

Reply:
We do, implementation changed. 

We don't need it per se, but I would prefer no crashes from the resource manager being null in an extension method if the test somehow doesn't go the right way.